internal/venice-test-common/src/main/java/com/linkedin/venice/utils/TestWriteUtils.java

```diff
@@ -865,16 +866,24 @@ public static Properties defaultVPJPropsWithD2Routing(
     props.put(VENICE_DISCOVER_URL_PROP, String.format("d2://%s", controllerServiceName));
     props.put(SOURCE_GRID_FABRIC, childRegionNamesToZkAddress.entrySet().iterator().next().getKey());
 
+    props.putAll(pubSubClientConfigs);
+    props.put("PubSubClientProperties", pubSubClientConfigs);
```

Comment:
why do we need to do `props.put("PubSubClientProperties", pubSubClientConfigs);`